tox.ini

```diff
@@ -18,7 +18,7 @@ deps =
   pytype ; python_version < '3.13'
 commands =
   pytest {posargs}
-  ["$PYTHON_VERSION" -lt "3.13"] && pytype vulture/core.py
+  py{38,310,311,312}: pytype vulture/core.py
```

Comment:
Was there a specific reason to exclude 39?
3.8 will be EOL in a month, but 3.9 in a year 😅

Reply:
For some reason tox stumbles over py39. Python 3.9 is sill tested via GitHub Actions.